src/transformers/tokenization_utils.py

```diff
@@ -414,6 +399,18 @@ def _from_pretrained(cls, pretrained_model_name_or_path, *init_inputs, **kwargs)
 
             raise EnvironmentError(msg)
 
+        if all(full_file_name is None for full_file_name in resolved_vocab_files.values()):
+            raise EnvironmentError(
+                "Model name '{}' was not found in tokenizers model name list ({}). "
+                "We assumed '{}' was a path, a model identifier, or url to a directory containing vocabulary files "
+                "named {} but couldn't find such vocabulary files at this path or url.".format(
+                    pretrained_model_name_or_path,
+                    ", ".join(s3_models),
+                    pretrained_model_name_or_path,
+                    list(cls.vocab_files_names.values()),
+                )
+            )
+
```

Comment:
Won't this break if one of the required files is absent but others are present?

Reply:
yeah, but it was mostly already the case before. We could have a notion of required vs. optional files at some point (or probably better, just one single file)

Reply:
Yes, let's probably transition progressively to a single serialization file handled by `tokenizers`.